src/Umbraco.Web.Common/Extensions/ImageCropperTemplateCoreExtensions.cs

```diff
@@ -558,7 +558,7 @@ public static class ImageCropperTemplateCoreExtensions
         }
 
         var cacheBusterValue =
-            cacheBuster ? mediaItem.UpdateDate.ToFileTimeUtc().ToString(CultureInfo.InvariantCulture) : null;
+            cacheBuster ? mediaItem.UpdateDate.ToFileTimeUtc().ToString("x", CultureInfo.InvariantCulture) : null;
```

Comment:
Instead of using a decimal digit for the cache buster value (that always had a lot of trailing zeros, because the update date only has a resolution of 1 second, but the filetime value 100 nanoseconds), I've changed this to a hexadecimal value to make it a lot shorter.